apps/system/style/app_titlebar.css

```diff
@@ -19,7 +14,7 @@
   background-position: bottom right;
 }
 
-#screen:not(.rocketbar-focused):not(.utility-tray):not(.locked) #statusbar.light,
+#screen:not(.rocketbar-focused):not(.utility-tray):not(.locked) .appWindow.active.light > .titlebar .statusbar-shadow,
```

Comment:
I think we need to remove line 18 though. The '>' is important in this case for applying the filter to the correct shadow (imagine windows inside windows like the activities).

Good catch! Thanks!